keras_nlp/models/albert/albert_masked_lm_preprocessor_test.py

```diff
@@ -26,24 +25,14 @@
 
 class AlbertMaskedLMPreprocessorTest(TestCase):
     def setUp(self):
-        vocab_data = ["the quick brown fox", "the earth is round"]
-        bytes_io = io.BytesIO()
-        sentencepiece.SentencePieceTrainer.train(
-            sentence_iterator=iter(vocab_data),
-            model_writer=bytes_io,
-            vocab_size=12,
-            model_type="WORD",
-            pad_id=0,
-            unk_id=1,
-            bos_id=2,
-            eos_id=3,
-            pad_piece="<pad>",
-            unk_piece="<unk>",
-            bos_piece="[CLS]",
-            eos_piece="[SEP]",
-            user_defined_symbols="[MASK]",
+        self.tokenizer = AlbertTokenizer(
```

Comment:
Same comments as above for all other models.

Reply:
Done.